src/fqs/test_fqs.py

```diff
@@ -0,0 +1,303 @@
+from typing import Any, Callable, FrozenSet, Set
```

Comment:
```suggestion
#!/usr/bin/env python3
from typing import Any, Callable, FrozenSet, Set
```

Reply:
More flexible execution handled in [094e344](https://github.com/Geopipe/fqs/pull/1/commits/094e344674ae5afb774364f1ccfba4b1893453ad).